### DISCUSSION TITLE
Beatmap offset slider can duplicate carousel items on top of each other

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#17171] [Q&A] [answered]
While checking https://github.com/ppy/osu/pull/17162, I stumbled upon this bug that I hope isn't too hard to reproduce

1. Move the beatmap offset slider, then quickly back out
2. Press on the carousel item right when a new one pops over it, and it should stay
3. Click on another beatmap above or below and witness the shenanigans
    - Scrolling using arrow keys right after activating the bug can fling you far away, which breaks the bug because the carousel items unload

https://user-images.githubusercontent.com/36758269/157279513-a33bc7ed-e7a7-451a-9020-a5aa8178e4e0.mp4

I managed to reproduce on the latest release, so the issue isn't caused by the pr, but probably related to the saving of the offset or something

https://user-images.githubusercontent.com/36758269/157279699-8762df9d-84f3-45db-acc6-c6712d99452c.mp4

[logs from the above video](https://github.com/ppy/osu/files/8207876/logs.zip)

Some unhandled exceptions may also occur if you mess around with it too hard, which you can see in the logs

###### tbh it's very fun to mess around with

Here's me getting it consistently with multiple beatmaps

https://user-images.githubusercontent.com/36758269/157288188-9c1e0e77-5a1b-4ce0-93fa-1a6130f914ea.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/17182

**Comment by peppy** [comment]
@Walavouchey can you provide your `client.realm`? It's weird that you can reproduce these bugs but we can't, so there's gotta be something causing a large overhead in realm operations.

**Reply by Walavouchey** [reply]
\<file removed\>

wasn't able to reproduce after removing `client.realm` and was able to again when i moved it back
